Added documentation for the `--command` flag (#279)
okteto/docs: src/content/reference/cli.mdx

| _--file_      | (string) | The path to the manifest file (default "okteto.yml")                                                 |
| _--namespace_ | (string) | The namespace to use (defaults to the current okteto namespace)                                      |
| _--context_   | (string) | The context to use (defaults to the current okteto context)                                          |
| _--env_       |  (list)  | Set environment variable in the dev container.                                                       |
| _--deploy_    |  (bool)  | Force execution of the commands in the `deploy` section of the okteto manifest (defaults to `false`) |
| _--reset_     |  (bool)  | Resets the file synchronization service. Use it if the file synchronization service stops working.   |

> | _--command_   |  (list)  | Provide additional commands to override the ones specified under the dev field in the manifest       |

Reviewer: ```suggestion
| _--command_   |  (list)  | Override the command specified in the `dev` section of the manifest |
```

Reviewer: Let me know  if you agree @abs007 ?

Author: This seems good. I'll apply the changes
